real_time/arduino_src/radio_buggy_mega/fingerprint.py

```diff
@@ -1,7 +1,7 @@
-""" fingerprint.py: 
```

Comment:
This raise the issue of trailing white spaces all over the repo. What do we do with them?
We have the following options:
* changing them, adding a diff, making the blame command more complicated to use
  * by making a "clean-up" pull request with tons of this white spaces removal
  * or doing like Vivaan is doing now. Removing them as he sees them, even though it is unrelated to the diff.
* keeping them in the long term. Removing them only if we work on the related lines. Less diffs, less pretty code

About the diffs number, if he reverts his changes on those line and we keep track of all the history on master, it is still going to keep the diffs, making the blame command less useful. We might think about merging branches by collating diffs into a single one (for small changes like this branch).